### ISSUE TITLE
InteractableVehicle.fuel is not updated on the server side, when you use VehicleManager.SendVehicleFuel

### ISSUE THREAD

**Issue by MidnightTokyo** [SmartlyDressedGames/Unturned-3.x-Community#3081]
After using VehicleManager.SendVehicleFuel InteractableVehicle.fuel on the server side it still returns the old value.
Using VehicleManager.SendVehicleFuel together with InteractableVehicle.askFillFuel solves this problem. What's wrong with the new netcode?

**Comment by MidnightTokyo** [comment]
Btw, I want to get a beret, if it's not too much trouble
https://steamcommunity.com/profiles/76561198293693375/

**Comment by rube200** [comment]
First, Nelson is not distributing berets anymore.
Second the method are looking at is `InteractableVehicle.askFillFuel`.
Vehicle manager.SendFuel is just used to send the fuel the vehicle have to players

**Comment by MidnightTokyo** [comment]
Sounds like a cop-out when the method sends fuel specifically to the car, and it's done for the client side

**Comment by MidnightTokyo** [comment]
For example, sendVehicleHealth works perfectly.

**Comment by rube200** [comment]
Exactly the method is just sending fuel to all players

**Comment by rube200** [comment]
Unturned have so consistency issues about methods

**Comment by MidnightTokyo** [comment]
Are you going to prove that was Nelson's idea?

**Comment by rube200** [comment]
Yes, for example to change the health of a vehicle u need to interact with it or shot it, so he uses vehicleManager, for fuel u just can change it interacting with vehicle itself (not talking about plugins), so interactavleVehicle

**Comment by MidnightTokyo** [comment]
Bruh why refactor the code when you can release Unturned II
